.github/workflows/build.yml

```diff
@@ -177,6 +192,8 @@ jobs:
           LLVM_VERSION: ${{needs.prepare.outputs.LLVM_VERSION}}
           MINGW_W64_VERSION: ${{needs.prepare.outputs.MINGW_W64_VERSION}}
           TAG: ${{needs.prepare.outputs.TAG}}
+          SOURCE_DATE_EPOCH: ${{needs.prepare.outputs.COMMIT_DATE_UNIX}}
+          BUILD_DATE: ${{needs.prepare.outputs.BUILD_DATE}}
```

Comment:
The `BUILD_DATE` doesn't seem to be applied here on the macos packaging - is that because the `tar` on macOS lacks support for the `--mtime` option?

Reply:
yes, using gnu-tar in the new revision